docs/skopeo-copy.1.md

```diff
@@ -58,6 +58,10 @@ If the authorization state is not found there, $HOME/.docker/config.json is chec
 
 Existing signatures, if any, are preserved as well.
 
+**--dest-compress-format** _format_ Specifies the compression format to use.  Supported values are: `gzip`, `bzip2`, `xz`, `zstd`.
```

Comment:
A problem with using xz and bzip2 is that those are not reflected in the manifest. Once https://github.com/containers/image/pull/563 is merged, it will actually fail during manifest update/conversion. I would prefer to not mention them.

Reply:
psuhed a new version without `xz` and `bzip2`